packages/onboard-react/src/components/HardwareWalletSteps/AddressListTable.tsx

```diff
@@ -55,16 +61,32 @@ export const AddressListTable: FC<AddressListTableProps> = ({
     [selected, onAccountSelected],
   );
 
+  const onPageChange = useCallback(
+    async (page: number) => {
+      const list = await selectAccountOptions.scanAccounts({
+        derivationPath,
+        chainId: chain.id,
+        asset,
+        start: page === 0 ? 0 : (page - 1) * perPage,
```

Comment:
Currently, second page shows same wallets as initial one

```suggestion
        start: page * perPage - perPage,
```

And change `const [page, setPage] = useState(0);` to `const [page, setPage] = useState(1);` (make default page 1)

Reply:
I cannot do this because `Pagination` component indexes from `0` so it would show page `2` after the initial load. However, I'll fix the condition so the second page is correctly displayed.